### DISCUSSION TITLE
Enabling "Update" tab for no specific roles

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#15544] [How to] [answered]
Today I enabled the module `ActivityRegister `and enabled `Modules - Track Changes / History` for that module. For all other modes I have `Modules - Track Changes / History` disabled, as I do not need the functionalty.

When viewing a record (or a list) as administrator, I see the change history:
![admin](https://user-images.githubusercontent.com/10230144/146013892-5d4f5253-df7f-4cec-a787-3db2c43b103a.jpg)

But when viewing the same record (or a list) as normal user, I do not see the change history:
![user](https://user-images.githubusercontent.com/10230144/146013949-8efc6e1f-95df-441b-bcd0-254f6dd76eb3.jpg)

I believe this is due to the way I have configured my system, but I currently cannot find what I would have to change to make the `Updates `tab available for all (a given `profile`). Has anyone an Idea where I should look further?

**Comment by rskrzypczak** [comment, marked as answer]
Check permissions on profiles:

![image](https://user-images.githubusercontent.com/10221999/146016374-aa01e37e-c580-44dd-9f88-efe1fc7b277a.png)

**Reply by diderich** [reply]
Thanks!
